specification/sbi_atee.adoc

```diff
@@ -10,8 +39,1238 @@ shown in the figure above, the TEE Runtime (operating in VS-mode) supports
+    /* The TVM encountered a host-mode timer interrupt (informational only; non-resumable). */
+    VIRTUAL_SUPERVISOR_TIMER = 6,
+    /* The TVM encountered a machine-mode timer interrupt (informational only; non-resumable). */
+    MACHINE_TIMER = 7,
+    /* The TVM encountered a user-mode interrupt (informational only; non-resumable). */
+    USER_EXTERNAL = 8,
+    /* The TVM encountered a supervisor-mode interrupt (informational only; non-resumable). */
+    SUPERVISOR_EXTERNAL = 9,
+    /* The TVM encountered a host-mode interrupt (informational only; non-resumable). */
+    VIRTUAL_SUPERVISOR_EXTERNAL = 10,
+    /* The TVM encountered a machine-mode interrupt (informational only; non-resumable). */
+    MACHINE_EXTERNAL = 11,
+    /* The TVM encountered a supervisor-mode interrupt (informational only; non-resumable). */
+    SUPERVISOR_GUEST_EXTERNAl = 12,
+};
+-------
+
+[source, C]
+-------
+enum Exception {
+    /* The TVM executed a misaligned instruction (informational only; non-resumable). */
+    INSTRUCTION_MISALIGNED = 0,
+    /* The TVM encountered instruction fault (informational only; non-resumable). */
+    INSTRUCTION_FAULT = 1,
+    /* The TVM executed an illegal instruction (informational only; non-resumable). */
+    ILLEGAL_INSTRUCTION = 2,
+    /* The TVM executed a breakpoint instruction (informational only; non-resumable). */
+    BREAKPOINT = 3,
+    /* The TVM executed a misaligned load (informational only; non-resumable). */
+    LOAD_MISALIGNED = 4,
+    /* The TVM encountered a load fault (informational only; non-resumable). */
+    LOAD_FAULT = 5,
+    /* The TVM attempted to executed a misaligned store (informational only; non-resumable). */
+    STORE_MISALIGNED = 6,
+    /* The TVM encountered a store fault (informational only; non-resumable). */
+    STORE_FAULT = 7,
+    /* The TVM made an ECALL not directed at the host (informational only; non-resumable). */
+    USER_ENVCALL = 8,
+    /* The TVM made an ECALL not directed at the host (informational only; non-resumable). */
+    SUPERVISOR_ENVCALL = 9,
+    /*
+     * The TVM made an ECALL request directed at the host.
+     * The host should examine GPRs A0-A7 in the `tvm_vcpu_supervisor_gprs`
+     * area of the VCPU shared-memory region to process the ECALL.
+    */
+    VIRTUAL_SUPERVISOR_ENV_CALL = 10,
+    /* The TVM made an ECALL not directed at the host (informational only; non-resumable). */
+    MACHINE_ENVCALL = 11,
+    /* The TVM encountered an instruction page fault (informational only; non-resumable). */
+    INSTRUCTION_PAGE_FAULT = 12,
+    /* The TVM encountered a unservicable load page fault (informational only; non-resumable). */
+    LOAD_PAGE_FAULT = 13,
+    /* The TVM encountered a unservicable store page fault (informational only; non-resumable). */
+    STORE_PAGE_FAULT = 15,
+    /* The TVM made an ECALL not directed at the host (informational only; non-resumable). */
+    GUEST_INSTRUCTION_PAGE_FAULT = 20,
```

Comment:
comment is wrong

Reply:
@abrestic-rivos What's the difference between a GUEST_INSTRUCTION_PAGE_FAULT and INSTRUCTION_PAGE_FAULT?

Reply:
Guest instruction page fault: 2nd-stage page fault on instruction fetch
Instruction page fault: 1st-stage page fault on instruction fetch

See comment above re: removing these and pointing at the privileged spec instead.